docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,10 +1,220 @@
 [[get-started-docker]]
 == Running the {stack} on Docker
 
-The Elastic Docker registry contains Docker images for all the products in the 
-{stack}. For more information, see:
+The Elastic Docker registry contains Docker images for all the products in 
+the {stack}: https://www.docker.elastic.co/
+
+[float]
+[[run-stack-docker]]
+=== Run with Docker Compose
+
+To get the default distributions of {es} and {kib} up and running in Docker, 
+you can use `docker-compose`:
+
+. Pull the docker images for {es} and {kib}:
++
+["source","sh",subs="attributes"]
+--------------------------------------------
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version}
```

Comment:
We can eliminate the explicit docker pull steps if we want given that the images are specified in the docker-compose files (and the version gets supplied when the command is run). I don't personally see any benefit in explicitly calling out the pull step and it would allow us to reduce the number of documented steps/user actions if we remove.